api/templates/design.go

```diff
@@ -927,7 +927,7 @@ var Input = Type("input", func() {
 		Meta(oneof.Meta, json.MustEncodeString([]*openapi.Schema{
 			{Type: openapi.String},
 			{Type: openapi.Integer},
-			{Type: "double"},
+			{Type: openapi.Number},
```

Comment:
Este som si nasiel jednu chybu, OpenApi nema typ `double`, pomylilo ma, ze nizsie (vid obrazok) mame `double`, ale to je iba string v enum, nie OpenApi typ.

![image](https://user-images.githubusercontent.com/19371734/190629734-8c1e4974-bd6a-41e8-acd2-e760cb9a983e.png)